docs/nomenclature.md

```diff
@@ -18,25 +27,81 @@ Mutant currently supports the following subjects:
 * Instance methods
 * Singleton (class) methods
 
-Other subjects (constants, class bodies for DSLs, ...) are possible but aren't
-implemented in the OSS version.
+Other subjects are possible (even project-specific subjects) but aren't implemented in the OSS version. Please get in touch with the authors. Some examples are:
+* Constants
+* Class bodies for DSLs
+
+The more subjects that mutant can alter in your project, the more mutations it can create, and so the higher your confidence you can have that your tests cover the semantics of your application. Please get in touch if you require subjects beyond those implemented in Mutant already - support may be available in the commercial version.
+
+## Mutation Operator
+
+A transformation applied to the AST of a subject. Mutant knows the following high level operator classes (you will not be tested on your knowledge of these terms)
+
+Please be aware that there is no reason to learn these terms (there’s no quiz at the end of this semester!) - we’re just providing them to give you some ideas of what high-level classes of changes mutant makes to your code. For a full list of all the Mutation Operators in mutant, see the code in the [meta directory](https://github.com/mbj/mutant/tree/master/meta).
+
+* **Semantic Reduction**
+
+    This type of transformation replaces a piece of code which has somewhat complex semantics with one that has simpler semantics. To aid understanding, here are a couple of different sub-categories you could put them into. 
+    
+    * **Method call replacement** - for example, `#==` -> `#eql?` -> `#equal?`
+
+        `#==` commonly performs conversion between types in addition to checking equality, while `#eql?`  tends to check only that the class and instance variables are equal. Therefore we would say that #== is semantically simpler.
+        
+        You could also think of a semantic reduction in these cases as an increase in “strictness” of the code. `#equal?` is a stricter equality test than `#eql?`, and so on
+        
+    * **Code removal** - For example, `def my_method; do_something; end` -> `def my_method; end`
```

Comment:
Might be clearer rendered as a multiline ```diff?

Reply:
Yeah. Probably would reduce the visual density of this section a bit too.. It is a bit hefty.